src/App.jsx

```diff
@@ -14,70 +15,80 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {(selectedGood === '')
+        && (
+          <h1 className="title is-flex is-align-items-center">
+            No goods selected
+          </h1>
+        )}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood.length > 0 && `${selectedGood} is selected`}
```

Comment:
```suggestion
      selectedGood && <h1 className="title is-flex is-align-items-center">
        `${selectedGood} is selected`}
```

Reply:
Now there are two h1 tags on the page and one of them is empty if there are no selected goods.